src/Native/build-native.cmd

```diff
@@ -40,21 +40,38 @@ goto :Arg_Loop
 :ToolsVersion
 :: Determine the tools version to pass to cmake/msbuild
 if not defined VisualStudioVersion (
-    if defined VS140COMNTOOLS (
+    if defined VS150COMNTOOLS (
+        call "%VS150COMNTOOLS%\VsDevCmd.bat"
+        goto :VS2017
+    ) else if defined VS140COMNTOOLS (
+        call "%VS140COMNTOOLS%\VsDevCmd.bat"
         goto :VS2015
     )
     goto :MissingVersion
 )
-if "%VisualStudioVersion%"=="14.0" (
+
+if "%VisualStudioVersion%"=="15.0" (
+    goto :VS2017
+) else if "%VisualStudioVersion%"=="14.0" (
     goto :VS2015
 )
 
 :MissingVersion
-:: Can't find VS 2015
-echo Error: Visual Studio 2015 required  
+:: Can't find VS 2015 or 2017
+echo Error: Visual Studio 2015 or 2017 required  
 echo        Please see https://github.com/dotnet/corefx/tree/master/Documentation for build instructions.
 exit /b 1
 
+:VS2017
+:: Setup vars for VS2017
+set __VSVersion=vs2017
+set __PlatformToolset=v141
```

Comment:
Yes, VS2017 supports installing either the `VC++ 2017 v141 toolset` or the `VC++ 2015.3 v140 toolset`. Given that the v141 is the default installed when selecting the `Desktop development with C++` workload, I opted for that one.

When/if we support dynamically locating VS2017 installs, we can actually query which packages are installed and set this based on whichever is available.